01 - Community/Obsidian Roundup/2021-12-25  Live Preview Updates and Documentation.md

```diff
@@ -0,0 +1,96 @@
+-   [ITS Theme](https://forum.obsidian.md/t/theme-its-dark-light-theme/12838/146) has been moved to SCSS and updated for Live Preview. There are a bunch of new Style Settings fixes and updates, and a new Nord color scheme.
+-   [Sandstorm](https://github.com/jaysan0/obsidian-sandstorm) and [Nebula](https://github.com/dlccyes/obsidian-nebula/) are both new.
+-   Here's a snippet that will make Obsidian look like [Apple Notes](https://discord.com/channels/686053708261228577/702656734631821413/923730545085665330). The creator is thinking about developing it out into a full-fledged theme; if that's something you'd be interested in let them know :)
+
+## Guides
+
+-   Here's a nifty guide on [how to design better links](https://uxdesign.cc/designing-better-links-for-websites-and-emails-a-guideline-5b8638ce675a), aimed at websites and emails but really useful for personal knowledge management.
+-   Here's how you can [remap buttons on an android phone](https://discord.com/channels/686053708261228577/864046194195431425/923283744956252202) so that you can, for example, use your "volume up" button to toggle the quick switcher when using Obsidian.
+
+## Discussions
+
+-   Here's a nice discussion over at Hacker News about [taking daily notes in plaintext](https://news.ycombinator.com/item?id=29661167), inspired by this lovely writeup about how Jeff Huang uses a [productivity text file](https://jeffhuang.com/productivity_text_file/)
+-   Here's a neat discussion [pushing back against collector's fallacy](https://discord.com/channels/686053708261228577/722584061087842365/914188614362071091).
+-   Here's an example of [how to organize academic PDFs](http://discordapp.com/channels/686053708261228577/722584061087842365/923326410116980826).
+-   A process for how to read/note/highlight articles and books, and how to process lit notes, plus some [tips for highlights and processing](https://forum.obsidian.md/t/how-do-you-read-process-material/29140/4?u=austin).
+-   Here's [a case for using spaced repetition to enhance personal knowledge management](https://thepuranik.home.blog/2021/12/24/second-brain-obsidian-spaced-repetition-plugin/).
+-   Here's an interesting discussion about [workflows to draw diagrams on the iPad with the Apple pencil](https://www.reddit.com/r/ObsidianMD/comments/rmgrt2/update_on_the_discussion_i_had_here_about_ipad/).
+-   Several people chimed in to help out [someone who was overwhelmed with the possibilities of Obsidian](https://www.reddit.com/r/ObsidianMD/comments/rk1un0/need_advice_too_overwhelmed_to_start_using_the/) and having trouble starting out.
+
+## Exemplars
+
+-   Here's a great example of [how to use Supercharged Links](https://discord.com/channels/686053708261228577/744933215063638183/923730166444851220) and color-coding of tags to flag files as action items visually in the file tree.
+
```

Comment:
Cosmetic: The spaces between the bullets in this Exemplars section mean its layout looks a little untidy compared to its neighbours:

![image](https://user-images.githubusercontent.com/4840096/149638035-91feddc7-30c5-4c9b-b9b0-56854ebca8b2.png)

Reply:
Let's add this to the first good issues category